javascript/organizing_your_javascript_code/classes.md

```diff
@@ -11,7 +11,7 @@ Since we've already gone fairly in-depth with Constructors, you don't have too m
 This section contains a general overview of topics that you will learn in this lesson.
 
 - Describe the pros and cons of using classes in JavaScript.
-- Briefly discuss how JavaScript's object creation differs from a language like Java or Ruby.
+- Briefly discuss how JavaScript's object creation differs from a language like Java.
```

Comment:
What do you think about rewording to the following or something similar? I think the LO and KC items would be better in a mire general frame. The current wording makes it feel like the point is to compare to Java, whereas the article in the KC part only uses Java as an example and focuses more on "JS is different to other things altogether"
```suggestion
- Briefly discuss how JavaScript's object creation differs from other object-oriented programming languages.
```